app/src/main/java/com/example/hoaht/myapplication/MainActivity.java

```diff
@@ -8,6 +8,11 @@
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        Test();
         setContentView(R.layout.activity_main);
     }
+
+    private void Test() {
```

Comment:
ERROR: [MethodNamingConventions] Method names should not start with capital letters
http://pmd.sourceforge.net/pmd-5.2.3/pmd-java/rules/java/naming.html#MethodNamingConventions